Sources/MapboxCoreNavigation/ReplayLocationManager.swift

```diff
@@ -68,24 +80,35 @@ open class ReplayLocationManager: NavigationLocationManager {
     }
     
     @objc internal func tick() {
-        guard let startDate = startDate else { return }
+        guard let startDate = startDate,
+              !locations.isEmpty else {
+                  return
```

Comment:
Should we message somehow that replaying is stopped due to empty locations? For example we could use delegate to report an error or 'pause' state.

Reply:
Too much hassle for such a simple case just for tests. I will add precondition.